index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <meta property="og:title" content="Linkbrary" key="title" />
+    <meta
+      name="description"
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+      key="description"
+    />
+    <meta
+      property="og:image"
+      content="./images/library.PNG"
+      key="image"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@linkbrary" />
+    <meta name="twitter:title" content="Linkbrary" />
+    <meta name="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta
+      name="twitter:image"
+      content="./images/library.PNG"
+    />
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+      <nav> 
+        <div class="toolbar">
+          <a href="index.html">
+            <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+          </a>
+          <a class="login" href="signin.html">
+            <span>로그인</span>
+          </a>
+        </div>
+      </nav> 
+
+
+        <div class="bg">
+            <div class="title">
+                <span class="grad">세상의 모든 정보</span><span class="black">를<br>쉽게 저장하고 관리해 보세요</span>
+            </div>
```

Comment:
페이지의 가장 중심 문구이기 때문에 h1의 역할로 보입니다!

```suggestion
            <h1 class="title">
                <span class="grad">세상의 모든 정보</span><span class="black">를<br>쉽게 저장하고 관리해 보세요</span>
            </h1>
```